#!/usr/bin/env python
PACKAGE = "openpose_ros_node_cfg"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()


gen.add("show_skeleton",    bool_t, 0,  "Boolean wether to show the openpose skeleton", True)
gen.add("show_bbox",        bool_t, 0,  "True to visualize bounding box around detected persons, show_skeleton must be true", True)


exit(gen.generate(PACKAGE, "openpose_ros_node", "openpose_ros"))